### ISSUE TITLE
"Collaborating with" section logos overflowing.

### ISSUE THREAD

**Issue by Krishna2323** [layer5io/layer5#3476]
#### Description
"Collaborating with" section logos overflowing.
![Screenshot 2022-11-28 171221](https://user-images.githubusercontent.com/85894871/204269679-0ce7928e-054b-4c76-8543-e5a94dbeb7f3.png)

#### Environment:
- Host OS: 
- Browser: 

---
<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="16px" align="left" /><h3> Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

**Comment by leecalcote** [comment]
Thanks! However, this is by design. The idea is to convey the notion that there are so many logos that we're not trying to show them all. A carousel (with no arrows and no scrollbar) could be added instead (or we can just close this issue).

**Comment by roottool** [comment]
I see. However in very small extent, the RedHat and VMware logo were completely hidden by the overflow. And this section is wrapped when it is under 1400px width.
https://github.com/layer5io/layer5/blob/2680995d4d75735d592cf2d82f07a837cf1d63a4/src/sections/Home/Partners-home/partnerSection.style.js#L13-L17

I have a suggestion. How about `max-width` changes from 1400px to grater than 1400px?

**Screenshots**

- 1401px width
  ![image](https://user-images.githubusercontent.com/11808736/204430698-c80a619c-fb7f-4a86-b78d-4a1f2d30883c.png)

- Check responsive in a browser dev tool
  ![responsive area](https://user-images.githubusercontent.com/11808736/204432889-6ef6ed23-4eb8-4a6f-bbc1-cd0c6964a9ca.gif)

**Environment**:

- Host OS: Windows 11
- Browser: Vivaldi 5.5.2805.48

**Comment by Nikhil-Ladha** [comment]
Hi @roottool, let's ignore the wrapping here and switch to an infinite carousel (without navigation arrows). We already use a slick carousel in the project at multiple places so, you can use the same package. But, do remember to add the `settings` option to the component such that the carousel is active only after a certain breakpoint. Before, we won't need it (at least not now).
